src/main/java/org/gridsuite/study/server/repository/timepoint/TimePointEntity.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server.repository.timepoint;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.gridsuite.study.server.networkmodificationtree.entities.TimePointNodeInfoEntity;
+import org.gridsuite.study.server.repository.StudyEntity;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * @author Le Saulnier Kevin <lesaulnier.kevin at rte-france.com>
+ */
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Entity
+@Builder
+@Table(name = "timepoint")
+public class TimePointEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private UUID id;
+
+    @ManyToOne
+    @JoinColumn(name = "studyUuid")
+    private StudyEntity study;
+
+    @OneToMany(orphanRemoval = true, mappedBy = "timePoint", cascade = CascadeType.ALL)
```

Comment:
@ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "study_id", foreignKey = @ForeignKey(name = "time_point_entity_fk"))
    private StudyEntity study;

Reply:
maybe should be better to add constraint name and fetchtype

Reply:
Done